src/subscription_manager/cli_command/attach.py

```diff
@@ -0,0 +1,256 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function, division, absolute_import
+
+#
+# Subscription manager command line utility.
+#
+# Copyright (c) 2010 Red Hat, Inc.
```

Comment:
We should probably update these while we're in here.

Reply:
I was going to add this to the global comment.  As of Python 3 the line:
+# -*- coding: utf-8 -*-
is no longer required, and there is no need to import from future with python 3